src/OpenTelemetry/BaseExporter.cs

```diff
@@ -132,5 +148,23 @@ protected virtual bool OnShutdown(int timeoutMilliseconds)
         protected virtual void Dispose(bool disposing)
         {
         }
+
+        /// <summary>
+        /// Called when the parent <see cref="TracerProvider"/> is set on the exporter.
+        /// </summary>
+        /// <param name="tracerProvider"><see cref="TracerProvider"/>.</param>
+        protected virtual void OnTracerProviderSet(TracerProvider tracerProvider)
+        {
+        }
+
+#if NET461 || NETSTANDARD2_0
+        /// <summary>
+        /// Called when the parent <see cref="OpenTelemetryLoggerProvider"/> is set on the exporter.
+        /// </summary>
+        /// <param name="logProvider"><see cref="OpenTelemetryLoggerProvider"/>.</param>
+        protected virtual void OnLogProviderSet(OpenTelemetryLoggerProvider logProvider)
```

Comment:
lets remove this in this PR./ we can always add it in the future, when the need arises.
Main concern is - LoggerProvider is not a strict Otel term, like TracerProvider.